verification/curator-service/ui/src/components/new-case-form-fields/Pathogens.tsx

```diff
@@ -0,0 +1,114 @@
+import { Field, useFormikContext } from 'formik';
+
+import { Autocomplete } from '@material-ui/lab';
+import NewCaseFormValues from './NewCaseFormValues';
+import React from 'react';
+import Scroll from 'react-scroll';
+import { TextField } from 'formik-material-ui';
+import axios from 'axios';
+
+export default function Pathogens(): JSX.Element {
+    return (
+        <Scroll.Element name="pathogens">
+            <fieldset>
+                <legend>Pathogens</legend>
+                <PathogensAutocomplete />
+            </fieldset>
+        </Scroll.Element>
+    );
+}
+
+// Based on https://material-ui.com/components/autocomplete/#asynchronous-requests.
+export function PathogensAutocomplete(): JSX.Element {
+    const [open, setOpen] = React.useState(false);
+    const [options, setOptions] = React.useState<Map<string, number>>(
+        new Map(),
+    );
+    const loading = open && options.keys.length === 0;
+    const { setFieldValue, setTouched, initialValues } = useFormikContext<
+        NewCaseFormValues
+    >();
+
+    React.useEffect(() => {
+        let active = true;
+
+        if (!loading) {
+            return undefined;
+        }
+
+        (async (): Promise<void> => {
+            const resp = await axios.get<string>(
+                'https://raw.githubusercontent.com/open-covid-data/healthmap-gdo-temp/master/suggest/pathogens.csv',
+            );
+            // CSV lines are of the form '123,Disease name' and we want to
+            // map that to [{'Disease name', 123}]
+            const retrievedOptions = new Map(
+                resp.data
+                    .split('\n')
+                    .map((option) => [
+                        option.substring(option.indexOf(',') + 1),
+                        +option.substring(0, option.indexOf(',')),
+                    ]),
+            );
+
+            if (active) {
+                setOptions(retrievedOptions);
+            }
+        })();
+
+        return (): void => {
+            active = false;
+        };
+    }, [initialValues, loading, setFieldValue, setOptions, setTouched]);
+
+    React.useEffect(() => {
+        if (!open) {
+            setOptions(new Map());
+        }
+    }, [open, setOptions]);
+
+    return (
+        <Autocomplete
+            multiple
+            filterSelectedOptions
+            itemType="string"
+            open={open}
+            onOpen={(): void => {
+                setOpen(true);
+            }}
+            onClose={(): void => {
+                setOpen(false);
+            }}
+            options={Array.from(options.keys())}
+            loading={loading}
```

Comment:
you're missing initialValue for when we edit a case

Reply:
That's defaultValue a couple lines down